lib/alpha_taxonomy/taxon_linker.rb

```diff
@@ -0,0 +1,23 @@
+require "csv"
+
+module AlphaTaxonomy
+  class TaxonLinker
+    def run!
+      ImportFile.new.grouped_mappings.each do |base_path, taxon_titles|
+        taxon_content_ids = taxon_titles.map do |taxon_title|
```

Comment:
Good suggestion. Intended to optimise this later if it ever presented an issue, but that might be an easy drop-in.